tests/lazy_lax_test.py

```diff
@@ -166,3 +199,25 @@ def test_concatenate(dim, base_shape, dtype, num_arrs, rng_factory):
   args = [rng(shape, dtype) for shape in shapes]
   op = lambda *args: lax.concatenate(args, dim)
   tu.check_lazy_fun(op, *args)
+
```

Comment:
FYI I've pretty much been copying and pasting tests from jax tests/lax_test.py, and in general I think that's a good approach. For example for transpose you could copy and paste testTransposeAgainstNumpy, and just change a few lines of the test to test fastar instead of lax :).

These tests look fine so I wouldn't bother changing this pr, just a tip for future ops.